src/main/java/net/neoforged/neoforge/client/model/generators/ModelBuilder.java

```diff
@@ -480,6 +480,24 @@ public ElementBuilder allFaces(BiConsumer<Direction, FaceBuilder> action) {
             return this;
         }
 
+
+         /**
+         * Modify all <em>possible</em> faces except provided in second argument dynamically using a function, creating new
+         * faces as necessary.
+         *
+         * @param action the function to apply to each direction
+         * @param exceptions directions which will be excluded from adding to model file
+         * @return this builder
+         * @throws NullPointerException if {@code action} is {@code null}
+         */
+        public ElementBuilder allFacesExcept(BiConsumer<Direction, FaceBuilder> action, Direction... exceptions) {
```

Comment:
Why not `Set<Direction> exceptions`? You're putting it into a Collection anyway. And it's not that much more to type for the caller `allFacesExcept(..., EnumSet.of(Direction.NORTH, Direction.SOUTH));`. It also means that the caller can keep their set in a static field and avoid having an Array and a List object created on every call. EnumSets also have a much faster contains() lookup than Lists.

Reply:
That's a good idea, I'll change it. I didn't think about it from the start